src/shared/utils/isTextInputElement.js

```diff
@@ -32,12 +33,18 @@ var supportedInputTypes = {
   'week': true,
 };
 
-function isTextInputElement(elem) {
+function isTextInputElement(elem: ?HTMLElement): bool {
   var nodeName = elem && elem.nodeName && elem.nodeName.toLowerCase();
-  return nodeName && (
-    (nodeName === 'input' && supportedInputTypes[elem.type]) ||
-    nodeName === 'textarea'
-  );
+
+  if (nodeName === 'input') {
```

Comment:
`instanceof` won’t work correctly with nodes from an `iframe` because it has its own `window` and thus its own `window.HTMLInputElement`.

http://stackoverflow.com/questions/26248599/instanceof-htmlelement-in-iframe-is-not-element-or-object